File: app/src/main/java/cn/ifafu/ifafu/data/entity/Menu.java
package cn.ifafu.ifafu.data.entity;

import android.content.Intent;
import android.graphics.drawable.Drawable;

public class Menu {

    private Drawable icon; //图标
    private String title; //标题
    private Intent intent;

    public Menu(Drawable icon, String title, Intent intent) {
        this.icon = icon;
        this.title = title;
        this.intent = intent;
    }

    public Drawable getIcon() {
        return icon;
    }

    public String getTitle() {
        return title;
    }

    public Intent getIntent() {
        return intent;
    }
}

File: app/src/main/java/cn/ifafu/ifafu/data/entity/Menu.kt
package cn.ifafu.ifafu.data.entity

import android.app.Activity
import android.content.Intent
import android.graphics.drawable.Drawable

data class Menu(
        val icon: Drawable, //图标
        val title: String,  //标题
        val activityClass: Class<out Activity>
)
